src/arrays/ArrayUtil.java

```diff
@@ -46,49 +80,84 @@ public static boolean contains(int[] array, int value) {
      * @return the first index of value in the array, if array does not contain given element return -1.
      */
     public static int indexOf(int[] array, int value) {
-
-        return 0;
+        // methodi pahanjy chisht em haskce-> ete "value"-n mer massivi mej ka uremn return massivi araji elementy??
+        for (int i = 0; i < array.length; i++) {
+            if (array[i] == value) {
+                // return array[0]; ete petq e veradardznenq araji index-i arjeqy
```

Comment:
I think you don't need so many comments in your code.
Before open your PR you should remove not important comments.